### PR TITLE
Set `VehiculeWheel3D` `suspension_travel` default value to a reasonable one

### LINKED ISSUE #75059: VehicleWheel3D suspension travel 5m (metres)
### Godot version

4.0

### System information

Any

### Issue description

This should be changed to something sensible like 0.5, the tooltip also suggests this in 4.0 thankfully but I think 5 entire metres is a bit of a stretch (ha) 

When I was a non-docs-reading noob, this caused me many hours of pain because i put my wheels next to my car expecting them to work but the car would always sink flat into the wheels or too over

### Steps to reproduce

Make a car visually ‘accurate’ with nodes, put the wheels in the right spot, don’t change any settings for anything, and then run the project to see your vehicle sink into the floor.

### Minimal reproduction project

n/a

### PR DESCRIPTION
**Pull request by adamscott**
Fixes #75059

Sets the `VehiculeWheel3D` `suspension_travel` default value to 0.2 meters. It is currently... 5 meters.

Can be cherry-picked for 4.0.1, I think.

### CODE REVIEW

**Comment by Calinou** [comment]
This is a compatibility-breaking change, so we should find a way to apply it to newly created projects only somehow.

**Comment by aaronfranke** [comment]
Considering that the default value is useless for most games, I think this is a perfectly acceptable behavior change for 4.1. But I would not cherry-pick to 4.0.x.

File: scene/3d/vehicle_body_3d.h [hunk 1]
```diff
@@ -50,7 +50,7 @@ class VehicleWheel3D : public Node3D {
 	Vector3 m_wheelAxleCS; // const or modified by steering
 
 	real_t m_suspensionRestLength = 0.15;
-	real_t m_maxSuspensionTravelCm = 500.0;
+	real_t m_maxSuspensionTravelCm = 20.0;
```
Comment (aaronfranke) [inline scene/3d/vehicle_body_3d.h:53]: Unrelated to this PR, but it's really weird that this isn't in meters like every other value.

**Review by aaronfranke — APPROVED** [review]

**Comment by YuriSizov** [comment]
Yeah, I agree, we can merge it without worrying much about the compatibility.

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**